### PR TITLE
Add and remove created

### PR DESCRIPTION
**Pull request by hislordshipprof**
- Created new JavaScript file for the new functionality.
- Implemented a function for adding a new task 
- Implemented a function for deleting a task
- Implemented a function for editing task descriptions.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          <svg
+            width="10"
+            height="10px"
+            viewBox="0 0 1024 1024"
+            xmlns="http://www.w3.org/2000/svg"
+            class="icon-enter"
+          >
+            <path
+              fill="#818183"
+              d="M864 170h-60c-4.4 0-8 3.6-8 8v518H310v-73c0-6.7-7.8-10.5-13-6.3l-141.9 112a8 8 0 0 0 0 12.6l141.9 112c5.3 4.2 13 .4 13-6.3v-75h498c35.3 0 64-28.7 64-64V178c0-4.4-3.6-8-8-8z"
```
Comment (MeqdamAlqudah) [inline src/index.html:49]: - [ ] Please make sure that hardcode functionality work fine 
image

the box beside the input field should be an input field also but with a type checkbox so the user can check the item and then remove it from the delete all complete button

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import './styles.css';
+
+const todo = [
```
Comment (MeqdamAlqudah) [inline src/index.js:3]: - [ ] Please make sure that you add a:- 
1- function for adding a new task (add a new element to the array) is implemented.
2- A function for editing task descriptions is implemented.
3- A function for deleting a task (remove an element from the array) is implemented.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  {
+    description: 'my daily task1',
```
Comment (MeqdamAlqudah) [inline src/index.js:5]: - [ ] Please all changes to the To-Do List should be saved in local storage.

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @hislordshipprof , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @hislordshipprof, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights

- [ ] All linters are passing :heavy_check_mark: 
- [ ] Meaningful pull request name :heavy_check_mark: 
- [ ] Descriptive pull request :heavy_check_mark:  

 ## Required Changes ♻️

- [x] Please as the previous reviewer mentioned, you need to add all functionalities: for adding, deleting, and editing a task to your list, so now I can't add a `to-do` to the list.

https://user-images.githubusercontent.com/74501523/178035818-8c1eafc1-8ab3-4937-8f89-8ed5a16b1e4f.mp4

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hislordshipprof** [comment]
> 

please to get it running you would have to start it using npm start when the server is started you will be able to see the work. But when you run it from the local server you will only see the above one you sent

**Comment by omar-labana** [comment]
# **Invalid Code Review Request ⚠️❌**

Hi @hislordshipprof 

While you made great effort in this project, unfortunately🚩, I cannot proceed to review your code😓.

Repo doesn't contain A README file representing the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by alexander16108 — APPROVED** [review]
Hi @hislordshipprof , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**